packages/core/echo/echo-schema/src/types.ts

```diff
@@ -54,14 +30,14 @@ export type ObjectMeta = S.Schema.Type<typeof ObjectMetaSchema>;
  * It is stricter than `T extends {}` or `T extends object`.
  */
 // TODO(burdon): Consider moving to lower-level base type lib.
-export type BaseObject<T> = { [K in keyof T]: T[K] };
+export type BaseObject = { [key: string]: any };
```

Comment:
move to live-object or remove TODO? also "T extends {}" comment since you've removed T

Reply:
I think the comment refers to the usage of base object: `T extends BaseObject` is stricter then `T extends {}`